db/migrate/20181229233811_remove_arrival_departure_from_registrations.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveArrivalDepartureFromRegistrations < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :registrations, :arrival, :datetime
```

Comment:
So do we plan to just remove this information, without migrating it to surveys?

Reply:
For my case, at LFNW, over ~2700 registrations on 2 events, we have 0 values recorded for travel info.

I don't see any value in writing a translation for that.

Reply:
In general, I don't agree with keeping the old data indefinitely anyway (and I think the GDPR agrees on this point).

One can always create a survey for a _future_ conference that collects the data... but what is the use case for keeping the old travel data?

Reply:
I honestly don't think anybody is using this information. I think it is fine to just removing without migrating it. We should though add this to the [CHANGES file](https://github.com/openSUSE/osem/blob/master/CHANGES.md) (we tend to forget to document changes :see_no_evil:)

Reply:
Added a note to CHANGES in bddfcd6.